kinto/core/utils.py

```diff
@@ -182,6 +182,13 @@ def dict_merge(a, b):
     return result
 
 
+def find_nested_value(dict, path):
+    if "." not in path:
```

Comment:
Unfortunately it is not as simple because path can contain dots: See what we did there: https://github.com/Kinto/kinto-admin/blob/62553505e3610831b6119780328dede6c0f784da/src/utils.js#L124-L165